yr.py

```diff
@@ -176,7 +176,7 @@ def __init__(self, filename):
 
     def _process_row(self, row):
         trans = Transaction()
-        trans.date = datetime.strptime(row["Date"], "%Y/%m/%d")
```

Comment:
Good point. I'll look into it. I had figured they simply changed their format over the years.